src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const populationsClass = document.querySelectorAll('.population');
+const totalPopulations = document.querySelector('.total-population');
+const averagePopulations = document.querySelector('.average-population');
+
+const copyPopulations = [...populationsClass]
+  .map(num => +num.textContent.replace(/,/g, ''));
```

Comment:
This code style looks better for me
```suggestion
const copyPopulations = [...populationsClass].map(
  num => +num.textContent.replace(/,/g, '')
);
```